Need to add in a "#include <sys/types.h>" for this erlang header file so
that a definition for ssize_t is found, which in turn allows the ejabberd
component to build successfully.

--- otp_src_R15B03/erts/emulator/beam/erl_driver.h.orig	2012-12-28 09:47:16.253855115 -0800
+++ otp_src_R15B03/erts/emulator/beam/erl_driver.h	2012-12-28 09:47:45.111231567 -0800
@@ -86,6 +86,7 @@
 
 #include <stdlib.h>
 #include <string.h>		/* ssize_t on Mac OS X */
+#include <sys/types.h>		/* ssize_t on Solaris */
 
 #if defined(VXWORKS)
 #  include <ioLib.h>
